### DISCUSSION TITLE
How do i style the loading spinner to be at the end of the button instead of the start of the button?

### DISCUSSION THREAD

**Discussion by lyqht** [saadeghi/daisyui#1178] [Q&A] [answered]
Here's a preview of a button where I added the `loading` classname.

![image](https://user-images.githubusercontent.com/35736525/192079094-a1a17b50-da47-456d-922b-a681ac8e34e8.png)

As I already have a image at the start of the button, I would prefer the loading spinner to be at the end of the button. 

Any help will be appreciated!

**Comment by saadeghi** [comment, marked as answer]
Currently, that loading spinner is not easy to customize because I don't want to bring unnecessary breaking changes, but for the positioning, you can add these classes: `before:order-2 before:ml-2`

https://play.tailwindcss.com/1gw9L8NXcM

Let me know if you have any questions

**Reply by lyqht** [reply]
works great, thank you!
